src/QueryEditor.tsx

```diff
@@ -149,22 +154,35 @@ export class QueryEditor extends PureComponent<Props> {
     return (
       <div>
         {options.query.rawQuery && (
-          <InlineFieldRow>
-            <InlineField
-              label="Cassandra CQL Query"
-              labelWidth={30}
-              tooltip="Enter Cassandra CQL query. Also you can use $__timeFrom and $__timeTo variables, it will be replaced by chosen range"
-              grow
-            >
-              <TextArea
-                placeholder={'Enter a CQL query'}
-                onChange={this.onQueryTextChange}
-                onBlur={this.props.onRunQuery}
-                value={this.props.query.target}
-              />
-            </InlineField>
-            <Button icon="pen" variant="secondary" aria-label="Toggle editor mode" onClick={this.onChangeQueryType} />
-          </InlineFieldRow>
+          <>
+            <InlineFieldRow>
+              <InlineField
+                label="Cassandra CQL Query"
+                labelWidth={30}
+                tooltip="Enter Cassandra CQL query. Also you can use $__timeFrom and $__timeTo variables, it will be replaced by chosen range"
+                grow
+              >
+                <TextArea
+                  placeholder={'Enter a CQL query'}
+                  onChange={this.onQueryTextChange}
+                  onBlur={this.props.onRunQuery}
+                  value={this.props.query.target}
+                />
+              </InlineField>
+              <Button icon="pen" variant="secondary" aria-label="Toggle editor mode" onClick={this.onChangeQueryType} />
+            </InlineFieldRow>
+            <InlineFieldRow>
+              <InlineField label="Alias" labelWidth={8} tooltip="Series name override. Plain text or template using column names, e.g. `{{ column1 }}:{{ column2}}`">
```

Comment:
I'd suggest to set labelWidth to 30 as in query field, it'll be visually better 

<img width="439" alt="image" src="https://github.com/HadesArchitect/GrafanaCassandraDatasource/assets/1742301/e02ab23b-17ab-48cc-b3db-94ca7d88e871">

Reply:
`Alias` word looks a bit short for such a large field:
<img width="550" alt="Screenshot 2023-10-12 at 11 49 01" src="https://github.com/HadesArchitect/GrafanaCassandraDatasource/assets/9631312/7619e7c3-44c9-4b8e-93f5-5fc69e0e1ed8">
Maybe we should rename `Alias` to `Legend Alias` or something like that? I believe it looks more neat:
<img width="527" alt="Screenshot 2023-10-12 at 11 52 49" src="https://github.com/HadesArchitect/GrafanaCassandraDatasource/assets/9631312/9b5cacfa-6e21-40c3-919b-50f00754316e">